Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator+Joining.swift

```diff
@@ -301,6 +301,26 @@ extension WebRTCCoordinator.StateMachine.Stage {
                 .stateAdapter
                 .didUpdateParticipants(participants)
 
+            let sessionId = await coordinator.stateAdapter.sessionID
+            context.participantsSubscriptionsAdapter = await .init(
+                sessionId: sessionId,
+                sfuAdapter: sfuAdapter,
+                participantsPublisher: coordinator
+                    .stateAdapter
+                    .$participants
+                    .map {
+                        $0
+                            .values
+                            /// We ignore the local participant as we don't need the SFU to tell us
+                            /// what info to render.
+                            .filter { $0.sessionId != sessionId }
+                            /// We sort to maintain simple order to avoid subscriptions updates when
+                            /// the order changes but callParticipants haven't been updated.
+                            .sorted { $0.sessionId < $1.sessionId }
```

Comment:
hm, but the order can impact the subscriptions. E.g. in spotlight, when the first participant changes, the track size will change. Similarly, when someone is not in the viewable area and starts speaking. Will those have an impact?